src/components/CreatePostInput/index.js

```diff
@@ -162,9 +149,9 @@
       resetTopicSearch();
       setPositionKeyboard('never');
       cancelAllRequest();
-      const removeCharacterAfterSpace = textSearch.split(' ')[0];
-      const newTopics = joinTopicIntoTopicList(removeCharacterAfterSpace, topics);
-      setTopics(newTopics);
+      // const removeCharacterAfterSpace = textSearch.split(' ')[0];
+      // const newTopics = joinTopicIntoTopicList(removeCharacterAfterSpace, topics);
+      // setTopics(newTopics);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

These lines have been commented out. If they are not necessary, consider removing them to improve readability. If they might be needed in the future, add a comment explaining why they're commented out.
```diff
-       // const removeCharacterAfterSpace = textSearch.split(' ')[0];
-       // const newTopics = joinTopicIntoTopicList(removeCharacterAfterSpace, topics);
-       // setTopics(newTopics);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->